drivers/d3d12/rendering_context_driver_d3d12.h

```diff
@@ -59,6 +59,20 @@
 #undef AS
 #endif
 
+#if WINVER < _WIN32_WINNT_WIN8
```

Comment:
```suggestion
#if (WINVER < _WIN32_WINNT_WIN8) && defined(_MSC_VER)
```

Seems like only MSVC headers have versions checks.

Reply:
Added the condition but not sure it's a good idea since the features are clearly marked as not being available <win8 and adding the msvc check makes it seem like that is only the case for visual studio.